test/e2e/scheduling/predicates.go

```diff
@@ -799,8 +804,72 @@ var _ = SIGDescribe("SchedulerPredicates [Serial]", func() {
 			framework.ExpectEqual(numInNode2, expected, fmt.Sprintf("Pods are not distributed as expected on node %q", nodeNames[1]))
 		})
 	})
+
+	ginkgo.It("validates Pods with non-empty schedulerGates are blocked on scheduling [Feature:PodSchedulingReadiness] [alpha]", func() {
```

Comment:
schedulingGates

Reply:
We need to add the feature gate to https://github.com/kubernetes/test-infra/blob/f0b00eafb2f8b9dfef40f3c2f9a8cb4b3bec5f75/config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml#L326 so that the test runs as part of the alpha job

Reply:
yup, I mentioned it in the PR description part.